tests/regression/tests/REQUEST-942-APPLICATION-ATTACK-SQLI/942550.yaml

```diff
@@ -0,0 +1,637 @@
+---
+meta:
+  author: "Andrea Menin (theMiddle)"
+  description: JSON in SQL bypass technique
+  enabled: true
+  name: 942550.yaml
+tests:
+  - test_title: 942550-1
+    desc: "JSON in SQL (ARGS)"
```

Comment:
I suggest you use a multiline string for the description and add the payload to the description, e.g.

```
desc: |
  JSON in SQL (ARGS)
  decoded payload: OR '{"b":2}'::jsonb <@ '{"a":1, "b":2}'::jsonb
```